File: task_1.py
import time

def number_test (quantity):
    results = []
    test_number = 1
    while len(results) < quantity:
        prime_number = True
        for i in range(2, test_number):
            if test_number % i == 0:
                prime_number = False
                break
        if prime_number == True:
            results.append(test_number)
        test_number += 1
    return results

number = int(input("Введите число: "))
start = time.time()
number_test(number)
end = time.time() - start
print(end)

File: task_1.js
console.time()
function checkNumber(quantity) {
    const results = [] // Массив результатов
    let testNumber = 1
    let primeNumber // Признак простого числа

    while (results.length<quantity) {
        primeNumber = true
        for (let index = 2; index < testNumber; index++) {
            if (testNumber % index === 0) {
                primeNumber = false; break
            }
        }
        if (primeNumber == true) {
            results.push(testNumber)
        }
        testNumber++
    }
    return results
}

console.log("Result:", checkNumber(Number(process.argv[2])))
console.timeEnd()